resources/todomvc/architecture-examples/angular/src/app/todo-item/todo-item.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, Input, Output, EventEmitter, ViewChild, ElementRef, AfterViewChecked, ChangeDetectionStrategy } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { Todo } from '../todo';
+
+@Component({
+  selector: 'app-todo-item',
+  templateUrl: './todo-item.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TodoItemComponent implements AfterViewChecked{
```

Comment:
nit: space before `{`

Is prettier running on this codebase?

Reply:
will check

Reply:
opted into root format 😄